app/assets/stylesheets/_variables.scss

```diff
@@ -91,6 +91,9 @@ $warning: $brand-yellow;
 $primary: $brand-blue;
 $dark: $brand-dark-blue;
 
+
+$brand-inverse: $brand-dark-blue;
+
```

Comment:
use existing `$dark` above, instead of defining a `$brand-inverse`. What Bootstrap 3 called `$brand-inverse`, Bootstrap 4 calls `$dark`.

Reply:
Cool that did work. (I was trying to do $brand-dark before).